workers/unity/Packages/io.improbable.worker.sdk/CIO.cs

```diff
@@ -30,8 +30,28 @@ protected override bool ReleaseHandle()
 
         public enum OpenMode
```

Comment:
As discussed offline, this could be changed to a `[Flags]` enum, typed at Uint32.